src/main/java/com/epam/ta/reportportal/core/events/activity/ProjectAnalyzerConfigEvent.java

```diff
@@ -57,8 +57,7 @@ public ProjectAnalyzerConfigEvent(ProjectAttributesActivityResource before,
 
   @Override
   public Activity toActivity() {
-    return configEquals(getBefore().getConfig(), getAfter().getConfig(), Prefix.ANALYZER) ? null
-        : convert();
+    return convert();
   }
```

Comment:
Maybe you can just move all the code to toActivity method instead of using convert()